aikau/src/main/resources/alfresco/pickers/SitePicker.js

```diff
@@ -0,0 +1,245 @@
+/**
+ * Copyright (C) 2005-2015 Alfresco Software Limited.
+ *
+ * This file is part of Alfresco
+ *
+ * Alfresco is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Alfresco is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/**
+ * @module alfresco/pickers/SitePicker
+ * @extends module:alfresco/pickers/Picker
+ * @author Dave Draper
+ */
+define(["dojo/_base/declare",
+        "alfresco/pickers/Picker"],
+        function(declare, Picker) {
+
+   var sitesView = {
+      name: "alfresco/lists/views/AlfListView",
+      config: {
+         widgets: [
```

Comment:
We probably also want to override the "noItemsMessage" message here to be site specific